src/pages/Committees.js

```diff
@@ -587,4 +641,204 @@ function SportsCommittee() {
+            </ul>
+        </p>
+    )
+}
+
+function MediaStudyBoard() {
+    return (
+        <p>
+            The Study Board (Swedish: Studienämnden) verifies that every student has a well-functioning education and safe
+            study environment. This applies at both programme-level and course-level. If you have any problem with courses and
+            programs, or have questions about them, you can contact the Study Board! Joining the committee lets you help everyone
+            have a better education and you will have a better understanding of education at KTH.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: sno@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function CommunicationCommittee () {
+    return (
+        <p>
+            The Communication Committee (Swedish: Kommunikationsnämnden) is the committee in charge of Media's external branding,
+            e.g. they update Media's Instagram, Facebook, etc. They also take care of the communications within the chapter and
+            announce events. Finally, they run a podcast on Spotify!
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: kommunikator@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function Branschdag () {
+    return (
+        <p>
+            Searching for a job after KTH? Media's carrier fair (Swedish: Branschdag) is Media's fair lets you find jobs,
+            degree project or simply to enrich your network, it's up to you! Don't miss it!
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: branschdag@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function BusinessGroup () {
+    return (
+        <p>
+            The Business Relations Group (Swedish: Näringslivsgruppen) initiate contacts with companies and provide students with opportunities
+            to build a strong network of business contacts. Working for the Business Group makes it doubly as effective for you. Members
+            organise events such as mingles or recruitement events.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: nlg@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function Festmästeriet () {
+    return (
+        <p>
+            Party people, here is your stuff! The Party Committee (Swedish: Festmästeri) is specialized in organizing funny and wholesome
+            parties and events. Members of the committee can do tasks as diverse as preparing the menu, doing PR (public relations, i.e.
+            advertisement), invent fun competitions for gasques or film gyckels. Join them to maximise the fun of Media's parties.
+            <br/>
+            Contact:
+            <ul>
+                <li>Mail: festmasteriet@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function FilmCommittee () {
```

Comment:
~~including a high-effort film project for each reception~~ the film project for the reception is the only thing they do, that's filmnämndens entire purpouse